public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -36,8 +36,8 @@ define(function(require) {
 
     // Update the UI with a "Saving..." indicator whenever we sync a file
     syncManager.on("file-sync-start", function() {
+	  $("#navbar-save-indicator").removeClass("hide");
       $("#navbar-save-indicator").text("{{ fileSavingIndicator }}");
```

Comment:
which code editor do you use? most let you say which indentation rules to use. At the very least they should has some kind of find/replace that lets you search for tabs (typically `\t`) and then replace it another string like `    ` (four spaces)